### DISCUSSION TITLE
Link ignored and blocked updates to the related PR

### DISCUSSION THREAD

**Discussion by allanlewis** [renovatebot/renovate#25848] [Suggest an Idea]
### Tell us more.

The "Ignored or Blocked" section of developer.mend.io lists updates that are being ignored because the PR was closed. On the dependency dashboard in GitHub Issues these entries are linked to the related PR, but not in the web app. It would be helpful if the web app had the same behaviour as the dashboard in this regard.

**Comment by PhilipAbed** [comment]
```
DEBUG: branches info extended
{
  "branchesInformation": [
    {
      "branchName": "renovate/azure-identity-1.x",
      "prNo": null,
      "prTitle": "Update dependency @azure/identity to v1.5.0",
      "result": "already-existed",
      "upgrades": [
        {
          "datasource": "npm",
          "depName": "@azure/identity",
          "displayPending": "",
          "fixedVersion": "1.0.0-preview.2",
          "currentVersion": "1.0.0-preview.2",
          "currentValue": "1.0.0-preview.2",
          "newValue": "1.5.0",
          "newVersion": "1.5.0",
          "packageFile": "package.json",
          "updateType": "minor",
          "packageName": "@azure/identity"
        }
      ]
    },
```

the extended info does not include the `prNo`, because by the time it reaches this print, the branch would be already deleted for a closed PR.
https://github.com/renovatebot/renovate/pull/31308
